kodi_addon_checker/check_dependencies.py

```diff
@@ -122,3 +135,19 @@ def _get_ignore_list(branch_name):
 
     else:
         return common_ignore_deps
+
+
+def check_extensions(report: Report, parsed_xml, addon):
+    """Check if required dependency exist with comparision to
+       the existing extension points
+
+      :addon: class kodi_addon_checker.addons.Addon.Addon
+
+    """
+    deps = [dependency.id for dependency in addon.dependencies]
+
+    for extension in parsed_xml.findall("extension"):
+        point = extension.get("point")
+        if point in extensions and extensions[point] not in deps:
+            report.add(Record(PROBLEM, "Required dependency {} is missing from the addons"
```

Comment:
This exact message is already used to point out that an dependency listed in addon.xml is not available the official Kodi repo.
Here it is used to point out that a dependency has to be added to addon.xml.

So one of those has to be repharsed to not cause confussion.